src/utils/helpers.ts

```diff
@@ -38,13 +38,22 @@ export type ThemeType = {
   padding?: SpacingSizeValue;
 };
 
+type HelperFn<Input, ReturnType> = {
```

Comment:
You can assing default value (`string`) to the second generic param and then you can omit it on other places. It should be relatively safe as we are returning only strings anyway.
```suggestion
type HelperFn<Input, ReturnType = string> = {
```